hoot-services/src/main/java/hoot/services/controllers/job/custom/HGIS/HGISResource.java

```diff
@@ -116,31 +80,29 @@ protected boolean _mapExists(final String mapName) throws Exception {
      * Creates db conection string based on config settings in
      * hoot-services.conf
      *
-     *
      * @param mapName
      * @return output looks like
      *         postgresql://hoot:hoottest@localhost:5432/hoot1/BrazilOsmPois
      */
     @SuppressWarnings("static-method")
-    protected final String _generateDbMapParam(final String mapName) {
-        return "hootapidb://" + getDbUserId() + ":" + getDbPassword() + "@" + getDbHost() + "/" + getDbName() + "/"
-                + mapName;
+    static String generateDbMapParam(String mapName) {
+        return "hootapidb://" + dbUserId + ":" + dbPassword + "@" + dbHost + "/" + dbName + "/" + mapName;
     }
 
     /**
      * Default parameter creator. Child class probably will override this to
      * meet its need. TODO: We would need to figure out the transport object and
      * replace jsonarray with it
      */
-    protected final JSONArray _createParamObj(final String in, final String out) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/DGIS/hootenanny/file/2578184282/issues/source?bid=3293092&fileBranchId=3301860#l97)